static/js/reducers/email.js

```diff
@@ -69,7 +69,8 @@ export const email = (state: AllEmailsState = INITIAL_ALL_EMAILS_STATE, action:
       params: action.payload.params || {},
       subheading: action.payload.subheading,
       supportsAutomaticEmails: action.payload.supportsAutomaticEmails,
-      filters: action.payload.filters
+      filters: action.payload.filters,
+      inputs: action.payload.inputs || NEW_EMAIL_EDIT,
```

Comment:
When we're editing an existing email we already have an `inputs` object defined, so we'd like to pass that in on the `payload`. Normally we start with a blank slate (the `NEW_EMAIL_EDIT` object).